network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/GeneratorAdderImpl.java

```diff
@@ -131,7 +131,7 @@ public Generator add() {
 
```

Comment:
No changes on this:
https://github.com/powsybl/powsybl-network-store/pull/483/files#diff-0ed70a6c3b8d9948ec0b2e055b4796c143ae86eae8acd6097d66ba37e5f28f0bR62-R67
?

Reply:
no it is in the adder

Reply:
read the class it is added when it is created in build() method